Rules/AvoidUsingInternalURLs.cs

```diff
@@ -44,8 +46,19 @@ public IEnumerable<DiagnosticRecord> AnalyzeScript(Ast ast, string fileName)
             {
                 foreach (StringConstantExpressionAst expressionAst in expressionAsts)
                 {
-                    //Check if XPath is used. If XPath is used, then we don't throw warnings.
+                    
                     Ast parentAst = expressionAst.Parent;
```

Comment:
I don't think it will ever be null..At least the root Ast should be a ScriptBlockAst right..